build.gradle

```diff
@@ -278,14 +278,16 @@ dependencies {
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-mail'
```

Comment:
can we try without this?

Reply:
done. We don't need starter-mail in orchestrator.